00_Base/src/ocpp/persistence/namespace.ts

```diff
@@ -9,6 +9,11 @@
 
 export enum Namespace {
   BootConfig = 'Boot',
+  ChargingProfile = 'ChargingProfile',
+  ChargingSchedule = 'ChargingSchedule',
+  ChargingNeeds = 'ChargingNeeds',
+  CompositeSchedule = 'CompositeSchedule',
+  SalesTariff = 'SalesTariff',
```

Comment:
`ChargingNeeds` and `SalesTariff` are both OCPP 2.0.1 specific object. I think we don't need to generalize them.